plone/app/upgrade/v52/tests.py

```diff
@@ -248,8 +248,8 @@ def test_migrate_markup_settings(self):
 
 def test_suite():
     # Skip these tests on Plone < 5.2a1
-    plone_version = get_distribution("Products.CMFPlone").version
-    if not parse_version(plone_version) >= parse_version("5.2a1"):
+    plone_version = distribution("Products.CMFPlone").version
+    if not version.parse(plone_version) >= version.parse("5.2a1"):
```

Comment:
This condition is actually always False. :-)  I will push a commit to remove these lines.

I think we can safely remove this whole `test_suite` function, and do that in two other test files as well, but let's not broaden the scope of this PR.